### ISSUE TITLE
Typescript: strange debugger behaviour on await functions

### ISSUE THREAD

**Issue by tonka3000** [microsoft/vscode#107573]
- VSCode Version: 1.49.2
- OS Version: Darwin x64 18.7.0
- ts-loader: 8.0.4
- webpack: 4.44.2

Steps to Reproduce:

1. clone https://github.com/tonka3000/vscode-qt-tools/pull/34
2. npm install
3. run the project via F5
4. go to any any await function call like https://github.com/tonka3000/vscode-qt-tools/blob/c0192046c08c9856b8f553138d3665419c2fe124/src/extension.ts#L77 and try to "step into" (F11) into the function. The debugger dont want to go into the function, I have to click multiple times to get anywhere. After many tries the debugger get to the right point, but it seems that the debugger hold on any generated js line.<br>
![VSCode-ts-loader-debugger-crazy_short](https://user-images.githubusercontent.com/3163807/94409365-5ba15300-0176-11eb-9014-d8e134a0c38d.gif)

I've try to port my extension to webpack/ts-loader to have a smaller bundle for performance reasons. I have just followed the instructions from https://code.visualstudio.com/api/working-with-extensions/bundling-extension. The debugger behaviour after the switch to webpack on await functions is really strange. On all other functions it works normally. I previously had the target es6 and the debugger jumps always into the js code (for whatever reason). After changing to es2019 target (hint from @connor4312 in the vscode slack channel) it got better, but the behaviour is still strange. I had no problem when I use typescript compiler only (but typescript only is no option for me).

I this an expected behaviour? I need webpack and ts-loader but the UX for VSCode extension development get really bad with the switch to webpackon async/await functions?

btw: smartStep is true.

Maybe someone here can help of any kind.

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by tonka3000** [comment]
I now added a typescript only (tsc) `launch.json` entry to my repo, but this does also not work. It seems that debugging typescript await/async functions is generally broken in VSCode now.

I have tried it on 3 machines (VSCode `1.49.3`), one MacBook, two Windows 10 Desktop machines, but on every machine there is the same problem. I've tried the [peacock repo](https://github.com/johnpapa/vscode-peacock) but I have the same problem.
